nifi-nar-bundles/nifi-html-bundle/nifi-html-processors/src/test/java/org/apache/nifi/AbstractHTMLTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi;
+
+import org.apache.nifi.flowfile.FlowFile;
+import org.apache.nifi.processor.ProcessSession;
+import org.apache.nifi.processor.io.StreamCallback;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class AbstractHTMLTest {
+
+    protected final String ATL_WEATHER_TEXT = "Atlanta Weather";
+    protected final String GDR_WEATHER_TEXT = "<i>Grand Rapids Weather</i>";
+    protected final String ATL_WEATHER_LINK = "http://w1.weather.gov/obhistory/KPDK.html";
+    protected final String GR_WEATHER_LINK = "http://w1.weather.gov/obhistory/KGRR.html";
+    protected final String AUTHOR_NAME = "Jeremy Dyer";
+    protected final String ATL_ID = "ATL";
+    protected final String GDR_ID = "GDR";
+
+    protected final String HTML = "<!doctype html>\n" +
```

Comment:
We should probably move this into a file under src/test/resources. Keeps the unit tests cleaner. Then we can easily enqueue it via runner.enqueue(new File("src/test/resources/XYZ.HTML").toPath());